src/types/jobs/schema.graphql

```diff
@@ -24,6 +24,10 @@ type Job {
     skills: [Skill]!
 }
 
- extend type Query  {
+# Interface and Union try
+union All = Job | Skill
+
+extend type Query {
     getJobById(id: ID): Job
+    getAll: [All]
```

Comment:
Keep in mind that you're evaluating each item of the array. even if it's an `interface` or `union`